bin/cnsadm

```diff
@@ -5,6 +5,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  *
  * Copyright (c) 2018, Joyent, Inc.
```

Comment:
This will give us build issues if you don't modify to `Copyright 2020 Joyent, Inc.` Mind to change that line too?